resources/config/common/simulation/primary_events/1.2/generators/dbd_more.def

```diff
@@ -0,0 +1,32 @@
+# -*- mode: conf-unix; -*-
+# List of multi-properties (datatools::multi_properties):
+
+#@description A list of DBD generators
+#@key_label  "name"
+#@meta_label "type"
+
+
+##############################################
+[name="Se82.2nubb_2MeV" type="genbb::wdecay0"]
+#@config Two neutrino double beta decay of Se-82, 2nubb;group=DBD/Se82
+#energy_sum_min=2~MeV
+decay_type      : string  = "DBD"
+decay_isotope   : string  = "Se82"
+decay_dbd_level : integer = 0
+decay_dbd_mode  : integer = 4 # 2nubb
+
+#@parameter The energy threshold on the electrons' energy sum
+energy_min      : real as energy = 2 MeV
+
+
+# #############################################
+# [name="Se82.2nubb_SSD" type="genbb::wdecay0"]
+# #@config Two neutrino double beta decay of Se-82, 2nubb;group=DBD/Se82;mode=SSD
+# decay_type      : string  = "DBD"
+# decay_isotope   : string  = "Se82"
+# decay_dbd_level : integer = 0
+# decay_dbd_mode  : integer = 8 # 2nubb (N*)
+
+
+
+# end of dbd_more.def
```

Comment:
This is not the same concept of versioning than in a VCS.
Here, this resource files "versioning" mechanism is used to "tag" various material that are used to build composite sets of resource files on demand, such sets being also tagged.
This mechanism allows to build a configuration setup by merging different sets of configuration resource files or elementary/atomic resource files. 
You can then build higher level setups by picking more fondamental resources from a hierarchy of tagged resources. This has nothing to do with VCS versioning. It is an identification system that allows to navigate through a  hierarchy/network of possibly dependant resources, organized in a space of topics, not time.
Of course, care must be taken to assemble together only compatible setups. But this has nothing to do with VCS  management. These validation work must be done even without a VCS. 
It is what we do when we prepare, as expert developers, a new MC configuration. VCS is not in the loop at this point yet,
but identification of geom/particles/vertex/G4 setup must be relevant.
With this system, it is possible to run in parallel several tagged setups within the same Git version number. Git (or whatever VCS)  manages timestamped software versions, while this mechanism manages tagged/identified conceptual setups, independantly of time.
IMO what is confusing is the semantic "X.Y[.Z]" of such tags which looks VCS semantic. 
I think I should write a note to explain these concepts that are not easy to address. 

Concerning the choice of "common/simulation/primary_events/1.2/generators/dbd_more.def"
of "snemo/demonstrator/simulation/primary_event/1.3" locations, it is completely arbitrary.
Our idea was to consider that:
- the "Se82.2nubb_2MeV" DBD generator can be used in general independantly of SuperNEMO simulations
  (BiPo...) so it is published as an extension of the existing set of "common" generators.
- the versatile generator is more specific demand related to the SuperNEMO MC group, it implies to create new variant
  stuff with consequence on the config system and the user interface. Anoter usage could have requested
another set of tweakable params for such generators. So the "snemo" config folder.
I admit that it is an editorial choice. Historically, Falaise was supposed to be used with other setups like BiPo, commissioning detector... HPGe... It is less relevant now.
Anyway, the point is to identify any new group of files with a proper unique tag, in such a way it can be freely used, linked, addresses, assembled at higher level without ambiguity.
HIH
Frc+Yv'

Reply:
@fmauger, I think this is actually an argument _for_ VCS! Specifically, it looks to be worthwhile exploring modularisation of the resource files so "tags" can become true Git tags. Creating new top level tags would then be a case of using something like submodules to create an overall bundle tag.

That's not for this PR, but please, contribute this type of info to #33!